crates/turborepo-scm/src/manual.rs

```diff
@@ -259,10 +281,17 @@ mod tests {
 
         let pkg_path = AnchoredSystemPathBuf::from_raw("child-dir/libA").unwrap();
         let unix_pkg_path = pkg_path.to_unix();
-        let file_hash: Vec<(&str, &str, Option<&str>)> = vec![
+        let mut file_hash: Vec<(&str, &str, Option<&str>)> = vec![
+            ("turbo.json", "turbo.json-file-contents", None),
+            ("package.json", "root-package.json-file-contents", None),
             ("top-level-file", "top-level-file-contents", None),
             ("other-dir/other-dir-file", "other-dir-file-contents", None),
             ("ignoreme", "anything", None),
+            (
+                "child-dir/libA/package.json",
```

Comment:
We should probably add a package-level `turbo.json` too.

Reply:
Good call, done!